src/markdown/CookiesPolicy.md

```diff
@@ -0,0 +1,32 @@
+## What Are Cookies
```

Comment:
Please remove this file, as we are not using right now.

Reply:
👍